bucket/git-credential-manager-core.json

```
@@ -0,0 +1,22 @@
+{
+    "version": "2.0.475",
+    "description": "Git Credential Manager Core (GCM Core) is a secure Git credential helper built on .NET that runs on Windows and macOS. Linux support is in an early preview.",
+    "homepage": "https://github.com/microsoft/Git-Credential-Manager-Core",
+    "license": "MIT",
+    "url": "https://github.com/microsoft/Git-Credential-Manager-Core/releases/download/v2.0.475/gcmcore-win-x86-2.0.475.64295.zip",
+    "hash": "498069454e7759364dd1c168a25d1f593d9295db521d4ec33e5897ce1344ea68",
+    "bin": "git-credential-manager-core.exe",
+    "shortcuts": [
+        [
+            "git-credential-manager-core.exe",
+            "GCM"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/microsoft/Git-Credential-Manager-Core",
+        "regex": "/releases/tag/v([\\d.]+)"
+    },
+    "autoupdate": {
+        "url": "https://github.com/microsoft/Git-Credential-Manager-Core/releases/download/v$version/gcmcore-win-x86-$version.$matchRandom.zip"
```

```suggestion
        "url": "https://github.com/microsoft/Git-Credential-Manager-Core/releases/download/v$matchHead/gcmcore-win-x86-$version.zip"
```
For details about built-in variables, such as **$matchHead** , check: https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate